### PR TITLE
Update UI translations for Tagalog, Korean, Mandarin, Spanish, Vietnamese, Amharic, and English

### PR DESCRIPTION
**Pull request by bion**
### Description

Updated translations for Tagalog, Korean, Mandarin, Spanish, Vietnamese, Amharic, and English

### CODE REVIEW

File: server/conf/messages.ko [hunk 1]
```diff
@@ -34,6 +34,12 @@ validation.isRequired=이 질문은 필수입니다.
 # Validation error that is shown when the user input couldn't be converted for any reason.
 validation.invalidInput=유효한 내용을 입력하세요.
 
+# Error message announced to screen reader when there are errors on the current page.
+validation.errorAnnouncementSr=양식에 오류가 있습니다. Please fix before continuing.
```
Comment (gwendolyngoetz) [inline server/conf/messages.ko:38]: NB: Did you mean to have the "Please fix..." English part here?

**Review by gwendolyngoetz — APPROVED** [review]
NB: Is it expected that some of these have more additions than others? Otherwise LGTM.

File: server/conf/messages.ko (hunk 1 above)
Comment (bion) [reply server/conf/messages.ko]: Nope, good catch!

**Comment by bion** [comment]
@gwendolyngoetz 
> NB: Is it expected that some of these have more additions than others? Otherwise LGTM.

Yep, some of the languages were missing more translations than others

**Review by gwendolyngoetz — APPROVED** [review]

**Pull request merged**